src/GovUk.Education.ExploreEducationStatistics.Content.Services/ThemeService.cs

```diff
@@ -130,81 +128,6 @@ private static PublicationTreeNode BuildPublicationNode(Publication publication)
             };
         }
 
-        // TODO: EES-2365 Remove once 'Download latest data' page no longer exists
-        [BlobCache(typeof(PublicationDownloadsTreeCacheKey))]
```

Comment:
We no longer need to bother invalidating this cache item now in the Admin on the Manage Publication page, when editing Themes and Topics, and during both workflows of the publisher. I think cleaning all that up will then leave `PublicationDownloadsTreeCacheKey` unused so it can be deleted after that.

Reply:
Also we could make a note to delete `publication-downloads-tree.json` from the environments after the deploy so it doesn't cause any confusion later on.

Reply:
Nice catch :smile: removed!